whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/WhoisRdapQueryLimitTestIntegration.java

```diff
@@ -1,35 +1,51 @@
-package net.ripe.db.whois.api.rdap;
```

Comment:
All test cases uses ES and are not covered by another ES test class.
So I prefer to move it into elasticsearch package instead of moving all the tests to WhoisRdapElasticServiceTestIntegration